src/main/java/com/github/paweladamski/httpclientmock/RuleBuilder.java

```diff
@@ -3,14 +3,17 @@
 import com.github.paweladamski.httpclientmock.action.Action;
 import com.github.paweladamski.httpclientmock.condition.Condition;
 import com.github.paweladamski.httpclientmock.condition.HttpMethodCondition;
+import com.github.paweladamski.httpclientmock.condition.UrlEncodedFormCondition;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import org.hamcrest.Matcher;
 
 class RuleBuilder {
```

Comment:
You forgot to add UrlEncdodedFormCondition to method `toRule`

Reply:
I add it using `RuleBuilder#addCondition`, so it is inside of the `conditions` list.